lib/tic_tac_toe.rb

```diff
@@ -23,4 +30,13 @@ def display_welcome
   def display_board
     @user_interface.display_board @board.board
```

Comment:
`board.board` smells a bit funny to me. An additional thing to consider is if we want to expose the underlying data structure of our board to the world. Does this break encapsulation? Is there a way we could hide the implementation of the board and still render and manipulate it?

Reply:
Having `to_s`, as mentioned [above](https://github.com/k3ntako/super-tic-tac-toe/pull/7/files#r380664733), would eliminate the need for having the `board.board` (needs to be renamed) exposed.

Reply:
I created a getter called state, so now the state of the board would be retrieved by `board.state`.

However, because formatting and displaying the board happens outside of the Board class, I think I would need to expose the state of the board.

Reply:
There are strategies we can employ to avoid exposing the board state explicitly but we can talk about those today!